.github/workflows/e2e_aks_runtime.yml

```diff
@@ -40,6 +41,7 @@ jobs:
           cat <<EOF > justfile.env
           container_registry=${{ env.container_registry }}
           azure_resource_group=${{ env.azure_resource_group }}
+          default_platform=AKS-CLH-SNP
```

Comment:
We now should be able to remove the platform from the lines 57 and 91

Reply:
The second one is not overridden by the default, but the first one should be.